### ISSUE TITLE
Remove glyphicon not shown when using with jQuery 3.5.1

### ISSUE THREAD

**Issue by danielrutten** [danielfarrell/bootstrap-combobox#271]
I've just upgraded form jQuery 3.4.1 to 3.5.1 and the glyphicon-remove is no longer shown when I select from the drop down box. I think the problem is due to the way the template is defined (see the template function on line 181).

It looks like 3.4.1 used to translate

`<span class="caret" /> <span class="glyphicon glyphicon-remove" />`

to

`<span class="caret"></span><span class="glyphicon glyphicon-remove"></span>`

but 3.5.1 turns that into

`<span class="caret"><span class="glyphicon glyphicon-remove"></span></span>`

Since the caret class is hidden when I select from the drop down box, the remove icon is hidden as wel.

I've fixed it for now by changing

`<span class="caret" /><span class="glyphicon glyphicon-remove" />`

into

`<span class="caret"></span><span class="glyphicon glyphicon-remove"></span>

but I was wondering if anyone else has found this problem.

**Comment by tiesont** [comment]
Yeah, spans are not self-closing, so that's not really valid. I'll see if I can fix that sometime today or this weekend.

**Comment by tiesont** [comment]
Related: https://jquery.com/upgrade-guide/3.5/#jquery-htmlprefilter-changes

**Comment by eeintech** [comment]
Thanks for that solution @danielrutten, works for me!
